vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -1,6 +1,10 @@
 from typing import Any, Dict, List, Optional
 
 import torch
+from compressed_tensors.config import CompressionFormat
```

Comment:
Is this equivalent to what we had before?
Causing test failures

Reply:
The tests seem to still installing the old `compressed-tensors==0.4.0`, which is missing the `naive_quantized` member